dashboard/streamlit_app.py

```
@@ -0,0 +1,83 @@
+st.title("CyberShell — ODS Evidence Dashboard")
+
+with st.sidebar:
+    st.header("Run Settings")
+    target = st.text_input("Target (lab/research only)", value="http://localhost:8000")
+    planner = st.selectbox("Planner", ["depth_first", "breadth_first"])
+    scorer = st.selectbox("Scorer", ["weighted_signal", "high_confidence", "default"])
+    llm_choice = st.selectbox("LLM", ["none", "openai", "ollama", "localfn"])
+    doc_root = st.text_input("Doc root", value="docs")
+    run_button = st.button("Run")
+
+def _make_llm(choice: str):
```

Ruff (0.12.2)

Missing return type annotation for private function `_make_llm`

(ANN202)